lib/embeds/tumblr.js

```diff
@@ -0,0 +1,31 @@
+import {render as renderEmbed} from 'embeds';
+import {renderString, tree} from 'deku';
+import loadEmbed from './load-embed';
+import renderEmbedIframe from './render-embed-iframe';
+
+export default {
+  name: 'TumblrEmbed',
+  render: ({props: {id}}) => {
+    return renderEmbedIframe({id, type: 'tumblr'});
+  },
+  afterMount: ({props}, iframe) => {
+    iframe.__props__ = props;
+    const {url, onLoaded, onResize} = props;
+    const content = renderString(tree(renderEmbed({type: 'tumblr', url})));
+    const protocol = window.location.protocol === 'file:'
+      ? 'https:'
+      : '';
+
+    const script = `<script src="${protocol}//assets.tumblr.com/post.js"></script>
+      <script>
+      var interval = setInterval(function () {
```

Comment:
@kesla, it's essentially the same, I just get in the habit of using `setTimeout` instead so you don't accidentally forget to clear intervals. looks like this:

```
function resizeTumblrEmbed() {
  var iframe = document.querySelector('.tumblr-embed');
  
  if (iframe.height) {
    resize();
  } else {
    setTimeout(resizeTumblrEmbed, 100);
  }
}
resizeTumblrEmbed();
```